Source/Core/VideoBackends/D3D/D3DBase.cpp

```diff
@@ -302,6 +303,28 @@ HRESULT Create(HWND wnd)
 											supported_feature_levels, NUM_SUPPORTED_FEATURE_LEVELS,
 											D3D11_SDK_VERSION, &swap_chain_desc, &swapchain, &device,
 											&featlevel, &context);
+		// Debugbreak on D3D error
+		if (SUCCEEDED(hr) && SUCCEEDED(device->QueryInterface(__uuidof(ID3D11Debug), (void**)&debug)))
+		{
+			ID3D11InfoQueue* infoQueue = nullptr;
+			if (SUCCEEDED(debug->QueryInterface(__uuidof(ID3D11InfoQueue), (void**)&infoQueue)))
+			{
+				infoQueue->SetBreakOnSeverity(D3D11_MESSAGE_SEVERITY_CORRUPTION, true);
```

Comment:
The DX control panel allows to override this setting, sure. But the same applies to the the Debug Creation Flag which is also enabled one line above.

This modifies just the default behaviour, since most of the time you are not specifically debugging Directx when crashes occur, and it would be nice to hit an int3 instead of crashing the driver.